src/net_processing.cpp

```diff
@@ -501,6 +501,14 @@ struct Peer {
     /** Whether this peer should be disconnected and marked as discouraged (unless it has the noban permission). */
     bool m_should_discourage GUARDED_BY(m_misbehavior_mutex){false};
 
+    /** Set of txids to reconsider once their parent transactions have been accepted **/
+    std::set<uint256> m_orphan_work_set GUARDED_BY(g_cs_orphans);
+
+    /** Protects m_getdata_requests **/
+    Mutex m_getdata_requests_mutex;
```

Comment:
Just realised you add this as `cs_vRecvGetData` in one commit and then change it with a scripted diff in the subsequent commit. You can just name it `m_getdata_requests_mutex` in the first commit!